scipy/stats/_new_distributions.py

```diff
@@ -198,6 +204,12 @@ def _moment_central_formula(self, order, **kwargs):
     def _moment_standardized_formula(self, order, **kwargs):
         return self._moment_raw_formula(order, **kwargs)
 
+    def _lmoment_formula(self, order, **kwargs):
+        lscale = 1 / np.sqrt(np.pi)
```

Comment:
See! If L-moments would have been the *standard* (i.e. normalized to have L-scale 1), the `1/sqrt(pi)` disappears form the normal PDF.  And then I wouldn't have had to cheat at my high school maths exams because the normal PDF was too complicated to remember (true story).